### DISCUSSION TITLE
how to use command ? in chat or instruct agent ?

### DISCUSSION THREAD

**Discussion by gandolfi974** [Josh-XT/AGiXT#537] [Q&A]
anyone can help me to use COMMAND ? example if i want to read or write a file. I have try a lot of prompt unsuccesfully in "chat" and "instruct agent". i don't know the difference.

```
- Command write_to_file executed successfully with args {'filename': 'poeme.txt', 'text': 'I saw an eagle in the sky today.'}
- Write to file 'poeme.txt' 'i saw an eagle in the sky today"
- 'command' : {'command_name' : 'Write to File', 'filename': 'poeme.txt', 'text': 'I saw an eagle in the sky today.'}
```
- Where is it save ?

thanks

**Comment by nightscape** [comment]
If you ran inside a Docker container, then the file is probably within that container.

**Reply by gandolfi974** [reply]
i start locally. There is no file in folder. I think the command is false and i don't know if i enter teh prompte in "Chat" or "Instruct agent"
i have tried another prompt unsuccesfully "  Command  "Write to File"   "filename": "poeme.txt"   "text" : "L'oiseau dans le ciel"      "
